src/ERC173/ERC173.sol

```diff
@@ -38,7 +38,22 @@ contract ERC173Facet {
     function transferOwnership(address _newOwner) external {
         ERC173Storage storage s = getStorage();
         if (msg.sender != s.owner) revert OwnableUnauthorizedAccount();
```

Comment:
Please change any `if` condition to using brackets `{` and `}`.   So change the above if statement to this:

```solidity
if (msg.sender != s.owner) {
    revert OwnableUnauthorizedAccount();
}